docs/development/overview.md

```diff
@@ -31,9 +31,7 @@ development environment:
 * On popular Linux platforms, you can **[install directly][install-direct]**.
 * On **other Linux/UNIX** distributions, you'll need to follow slightly different
   instructions to **[install manually][install-generic]**.
-
-Unfortunately, the only supported method to install on Windows is the [Vagrant
-method][install-vagrant].
+* On windows platform, you can **[install directly](https://zulip.readthedocs.io/en/latest/development/setup-advanced.html#installing-directly-on-windows-10-experimental)** via wsl.
```

Comment:
```suggestion
* On windows platform, you can **[install directly][install-via-wsl]** via wsl.
```

and add a relative link by using: 
```
[install-via-wsl]: ../development/setup-advanced.html#installing-directly-on-windows-10-experimental
```

Reply:
hey!! @aryanshridhar will you suggest me why my tests are failing , so that i would improve them.